QUEST.tsv

```diff
@@ -2098,156 +2098,156 @@ QUEST_20150714_002097	But, Raius is the disciple I acknowledge.{nl}His passion t
+QUEST_20150714_002194	$1. Dried environment{nl}2. Something that can be ridden on. {nl}3. Enough mana. {nl}4. The blessing of the Goddesses.
+QUEST_20150714_002195	$Flurry… turned into a handful of ashes by illuminating a great light. {nl}Did that demon do something? {nl}I had no idea, even as a probationary wizard.
+QUEST_20150714_002196	$[Caution]{nl}- It is the last stage of the experiment. Be cautious when handling it -
+QUEST_20150714_002197	$1. Do not overreact Schilt Essences into a sample of Petrification Detector. {nl}2. Do not keep the Schilt Amplifier activated for a long time. {nl}3. Even if you become immune to petrification using the Schilt Amplifier, it will only be temporary so do not stay in the lab room for too long.
+QUEST_20150714_002198	$[Usage Limit]{nl}- Due to a recent incident, we sealed the isolated lab room. {nl}Only people permitted by the chief can enter the room via the portal.
+QUEST_20150714_002199	$[Method of Usage]{nl}1. Use the Schilt Essences on the portal generator. {nl}2. One person can enter after a certain number of Essences are used. {nl}3. When more than one person enters simultaneously, please notify and ask the chief.
+QUEST_20150714_002200	$Petrification in Progress
+QUEST_20150714_002201	$My body is becoming stiff.
+QUEST_20150714_002202	$We can move faster.
+QUEST_20150714_002203	$We tried our best to impede them until your arrival. {nl}But as you can see, we're at our limit.
+QUEST_20150714_002204	$I've received your letter. {nl}Now is the chance the Goddesses gave to you since Legwyn is dead. {nl}It's great to hear that the lads of Legwyn family trusts you.
+QUEST_20150714_002205	$I've changed the name on this certificate for you so return the rest of documents to me. {nl}I guarantee either his assets or his land document. {nl}Please pay me the deposit as you did last time.
+QUEST_20150714_002206	$Anyone under the Legwyn name can't be left alive. Even the servants. {nl}Please also get rid of the daughter of Legwyn who is Squire Master.
+QUEST_20150714_002207	$Suspicious Man
+QUEST_20150714_002208	$It seems that you don't have a problem moving your body. {nl}You almost got into deep trouble.
+QUEST_20150714_002209	$Ah, I am called Mints. I led the hunters. {nl}If it's fine with you, I want to talk about at my camp located in Dumbala Intersections.
+QUEST_20150714_002210	$For now, there is no way to help the patient.
+QUEST_20150714_002211	$Have you tried giving him the antidote?
+QUEST_20150714_002212	$Please go to Vilties Forest after persuading the senior monk.
+QUEST_20150714_002213	$Thanks! {nl}I ran away without noticing my bag was ripped by the thorns.
+QUEST_20150714_002214	$We ran away from Raukime Swamp. {nl}When you find the pages of the documents from Ershike Trail to Dumbala Intersections, please bring them to me.
```

Comment:
Raukime Swamp is not in the wiki, but before we put a $ here, we should find out if the spelling is correct. Asked Xenobius on discord